src/Connection/Comments.php

```diff
@@ -73,7 +73,7 @@ public static function register_connections() {
 		/**
 		 * Register Connections from all existing PostObject Types to Comments
 		 */
-		$allowed_post_types = \WPGraphQL::get_allowed_post_types();
+		$allowed_post_types = \WPGraphQL::get_allowed_post_types( [ 'connect_to_comments' => true ] );
```

Comment:
How is this different than what we're already doing here: https://github.com/wp-graphql/wp-graphql/blob/2a349fdf8f4355590ddd2694e8ad4910d0ea07fe/src/Connection/Comments.php#L80

What if you set `connect_to_comments => true` but the post type doesn't register support for comments?

Seems kinda like an akward rule when we already have: `show_in_graphql` and `supports => [ 'comments' ]`

Reply:
@jasonbahl Not necessarily, currently I manual register the comments connection for the `Product` type in WooGraphQL so I can add the comment meta `rating` as an edge field.

Reply:
The Disqus use case is interesting. 🤔 

You would want the post_type to support comments in general, but would want to be able to disable the connection in GraphQL because the actual comments are handled outside of WordPress. 🤔

Reply:
that or in a custom table.